plugins/DualFilter/DualFilterControls.cpp

```diff
@@ -23,140 +23,131 @@
  *
  */
 
+#include "DualFilterControls.h"
 
 #include <QDomElement>
 
 #include "BasicFilters.h"
-#include "DualFilterControls.h"
 #include "DualFilter.h"
-#include "embed.h"
 #include "Engine.h"
 #include "Song.h"
+#include "embed.h"
 
-DualFilterControls::DualFilterControls( DualFilterEffect* effect ) :
-	EffectControls( effect ),
-	m_effect( effect ),
-
-	m_enabled1Model( true, this, tr( "Filter 1 enabled" ) ),
-	m_filter1Model( this, tr( "Filter 1 type" ) ),
-	m_cut1Model( 7000.0f, 1.0f, 20000.0f, 1.0f, this, tr( "Cutoff frequency 1" ) ),
-	m_res1Model( 0.5, BasicFilters<>::minQ(), 10.0, 0.01, this, tr( "Q/Resonance 1" ) ),
-	m_gain1Model( 100.0f, 0.0f, 200.0f, 0.1f, this, tr( "Gain 1" ) ),
-
-	m_mixModel( 0.0f, -1.0f, 1.0f, 0.01f, this, tr( "Mix" ) ),
-
-	m_enabled2Model( true, this, tr( "Filter 2 enabled" ) ),
-	m_filter2Model( this, tr( "Filter 2 type" ) ),
-	m_cut2Model( 7000.0f, 1.0f, 20000.0f, 1.0f, this, tr( "Cutoff frequency 2" ) ),
-	m_res2Model( 0.5, BasicFilters<>::minQ(), 10.0, 0.01, this, tr( "Q/Resonance 2" ) ),
-	m_gain2Model( 100.0f, 0.0f, 200.0f, 0.1f, this, tr( "Gain 2" ) )
+DualFilterControls::DualFilterControls(DualFilterEffect* effect)
+	: EffectControls(effect)
+	, m_effect(effect)
+	,
+
+	m_enabled1Model(true, this, tr("Filter 1 enabled"))
+	, m_filter1Model(this, tr("Filter 1 type"))
```

Comment:
There's something going on a little strange with these reformatted member initializer lists.
(Specifically the lone comma left on line 39, extra newline, and then the member on another newline.)

Reply:
Huh, could it be that it took exactly that line-spacing which was used for the members in the header file? I'll open a StackOverflow issue, probably.

Reply:
I just see that clang-format only tried to keep the original spacing in the CTOR initializer list. However, it does not cope well with the commas. I guess the only way to fix this is removing the empty lines between the CTOR initializer members. Is that acceptable?

Reply:
Resolving, due to inactivity and since it can be fixed by modifying the code. Feel free to comment against it and reopen.